instruments/qubitekk/mc1.py

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Provides support for the Qubitekk MC1 Motor Controller.
+
+MC1 Class originally contributed by Catherine Holloway.
+"""
+
+# IMPORTS #####################################################################
+from __future__ import absolute_import
+from builtins import range
+from enum import Enum
+import quantities as pq
+
+from instruments.abstract_instruments import Instrument
+# CLASSES #####################################################################
+from instruments.util_fns import int_property, enum_property, unitful_property
+
+
+class MC1(Instrument):
+    """
+    The MC1 is a controller for the qubitekk motor controller. Used with a
+    linear actuator to perform a HOM dip.
+    """
+    def __init__(self, filelike, increment=1, upper_limit=300,
```

Comment:
All these extra `__init__` parameters can't be there, and can't be used by the user anyways. Since we always initalize through the class methods `Instrument.open_X` all of these can't be used.